docs/diagrams/CommandsDiagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+!include Styles.puml
+
+Package "Commands                    "  <<Rectangle>>{
+     Class "{abstract}\nCommand" as Command COLOR_COMMANDS {
+     {method} + setData
+     {method} + setParams
+     {method} + execute: String
```

Comment:
Execute should be abstract, and since execute is abstract, for all the commands, need to show the execute method

Reply:
Also need brackets along with the type of parameters they take in

Reply:
IndexedCommand does not have a execute method actually. Its child classes inherit from Command class through IndexedCommand so I don't think need to include execute() in IndexedCommand.